front-end/src/Conf.ts

```diff
@@ -7,3 +7,9 @@ export const bluetoothBroadcastPeriodMilliseconds: number = 10000;
 export const bluetoothReconnection: boolean = false;
 // Clear browser local storage (Slowvid keys only) at startup
 export const clearLocalStorageOnStartup: boolean = true;
+
+export const backendAddr: string = "http://localhost:3002/api";
```

Comment:
Use the http router as above.
```suggestion
export const backendAddr: string = "http://localhost:3000/http";
```